dotcom-rendering/src/amp/lib/image-fit.ts

```diff
@@ -16,6 +16,7 @@ export const bestFitImage = (
 		.sort((a, b) => a.width - b.width)
 		.find((img) => img.width >= containerWidth);
 
+	// @ts-expect-error -- we’re checking the array is not empty above
```

Comment:
I gave up on rewriting this code as there's already a type-guard above, but TypeScript doesn't understand it.